sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -54,34 +55,32 @@
       "ky",
       "E"
     ],
-    "slow_axes": "@attrs:metadata/instrument/analyzer/slow_axes",
+    "slow_axes": "t.b.d.",
```

Comment:
The only possible slow axis for FLASH is probably only "delayStage" at the moment, or empty for static scans. This should ideally be filled programatically by the loader or calibration routines.

Reply:
Probably also in the case of spin-resolved measurements switching magnetic coil could be another slow axes.

Reply:
Maybe yes, spin-resolved ARPES is something still to develop completely.